scripts/xtensa-build-all.sh

```diff
@@ -60,6 +62,16 @@ while getopts "rudj:ck:va" OPTION; do
 done
 shift $((OPTIND-1))
 
+if [ -n "${OVERRIDE_CONFIG}" ]
+then
+	OVERRIDE_CONFIG="src/arch/xtensa/configs/override/$OVERRIDE_CONFIG.config"
+	if [ ! -f "${OVERRIDE_CONFIG}" ]
+	then
+		echo "Invalid override config file"
+		exit 1
+	fi
```

Comment:
Can you take this opportunity to copy/paste the (highly re-usable) `die()` function from `scripts/qemu-check.sh` instead?
```shell
die()
{
        >&2 printf '%s ERROR: ' "$0"
        # We want die() to be usable exactly like printf
        # shellcheck disable=SC2059
        >&2 printf "$@"
        exit 1
}
```
then
```shell
[ -f "${OVERRIDE_CONFIG}" ] ||
     die 'Invalid override config file %s\n' "${OVERRIDE_CONFIG}"
```